### ISSUE TITLE
Original Authors

### ISSUE THREAD

**Issue by moriarty** [rockin-robot-challenge/rockinYouBot#1]
The package.xml authors have been changed.

The original packages are distributed under GPLv3. I don't see any changes and I believe you the original authors should be credited, and the maintainer can be updated to Tim

**Comment by timfri** [comment]
This is correct. I asked Fred how we should handle this and we decided to put my name, so that any support questions will come to me from this repository.

**Comment by moriarty** [comment]
I think what is should be is:

Maintainer - the person to contact for support. 
Author - the original author

Or are we on a different page?
